Models/testmodel1/config.hpp

```diff
@@ -0,0 +1,133 @@
+#ifndef ZPSW3_CONFIG_HPP
+#define ZPSW3_CONFIG_HPP
+
+#include "global_config.hpp"
+#include "tim.h"
+
+namespace config
+{
+
+    /* Motor config */
+
+    constexpr Motor_t motors[] = {
+        {   //Yaw servo motor
+            .axis = yaw,
+            .isInverted = false,
+            .driverConstructor = constructDriver<MotorChannel, PWMChannel,
+                                                 /*timer*/ &htim1, /*timer_channel*/ 0>
+        },
+        {   //Roll BLDC motor
+            .axis = roll,
+            .isInverted = true,
+            .driverConstructor = constructDriver<MotorChannel, TempDSHOTDriver>
+        }
+    };
+
+    constexpr uint8_t NUM_MOTORS = sizeof(motors)/sizeof(Motor_t);
+
+
+
+    /* RC input config */
+
+    //Example array for a model that supports both PPM and SBUS input
+    constexpr RCInput_t RCInputs[] = {
+        {   //PPM input
+            .driverConstructor = constructDriver<RCInputDriver, TempPPMDriver>
+        },
+        {   //SBUS input
+            .driverConstructor = constructDriver<RCInputDriver, TempSBusDriver>
+        }
+    };
+    
+    constexpr uint8_t NUM_RC_INPUTS = sizeof(RCInputs)/sizeof(RCInput_t);
+
+
+
+    /* GPS config */
+
+    //Example array for a model with two GPS modules
+    constexpr GPS_t GPSArray[] = {
+        {   //NEOM8
+            .driverConstructor = constructDriver<GPSDriver, TempNEOM8Driver>
+        },
+        {   //Other GPS
+            .driverConstructor = constructDriver<GPSDriver, otherGPSDriver>
+        }
+    };
+    
+    constexpr uint8_t NUM_GPS = sizeof(GPSArray)/sizeof(GPS_t);
+
+
+
+    /* Flightmode Config */
+
+    class TempFlightmode1 : public AM::Flightmode{
```

Comment:
Are these meant to be defined in here or in the AttitudeManager/FlightModes/Inc?

Reply:
The idea behind having them here was that any reference to a flight mode that isn't compatible would be caught during compilation.